src/main/java/org/spongepowered/common/world/WorldManager.java

```diff
@@ -186,8 +188,25 @@ public static void registerDimensionType(int dimensionTypeId, DimensionType type
         return Optional.empty();
     }
 
-    public static Integer getNextFreeDimensionId() {
-        return dimensionBits.nextClearBit(0);
+    /**
```

Comment:
Remove these docs. It is not intended for plugins to call this code.

Reply:
Maybe not,  but it's good to document it for implementation (just like PhaseTracker is slowly being documented).

Reply:
These docs are focused on external code calling this method....